src/hardware/gus.cpp

```diff
@@ -233,6 +233,7 @@ class Gus {
 	virtual ~Gus();
 
 	bool CheckTimer(size_t t);
+	void MirrorAdlibCmdReg(const uint8_t reg_value);
```

Comment:
It would be better to spell it out

`MirrorAdLibCommandRegister`

We don't have to pay money for those extra letters 😎

Reply:
replacing!